tests/pytest_utils.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture
+def mat_url() -> str:
```

Comment:
This should not be a pytest fixture. Maybe a module variable in `utils.py` above instead.

We can also consider setting it up as an astropy configuration item. See https://docs.astropy.org/en/stable/config/#customising-config-in-affiliated-packages

Reply:
I've added `requests` temporarily until I can learn how to implement a the `astropy` configuration